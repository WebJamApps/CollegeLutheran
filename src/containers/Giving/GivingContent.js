import React from 'react';
import ELCALogo from '../../components/elcaLogo';

const pageName = 'giving';

const GivingContent = () => (

  <div className="page-content">
    <div className="container-fluid">
      <p style={{ fontSize: '4pt', margin: '0' }}>&nbsp;</p>
      <div className="material-content elevation3" style={{ maxWidth: '998px', paddingBottom: '-80px', margin: 'auto' }}>
        <h3 style={{ paddingTop: '22px', paddingBottom: '15px', fontWeight: '100' }}>Giving</h3>
        <p>
          College Lutheran Church now accepts online gifts through GivePlus and Vanco Payment Services.
          Giving with your credit card is easy. Just complete the form below to support the ministries of CLC with your offering!
        </p>
      </div>
      <p>&nbsp;</p>
      <div style={{
        margin: 'auto', textAlign: 'center', padding: '0', marginLeft: '-15px',
      }}
      >
        <iframe
          className="giving-iframe"
          src="https://gp.vancopayments.com/gpo/#/195e5d99-170c-4f3a-b5eb-e61ff13"
          title="Just Giving"
        >
          <p>Your browser does not support iframe.</p>
        </iframe>
      </div>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <ELCALogo pageName={pageName} />
      <p>&nbsp;</p>
    </div>
  </div>
);

export default GivingContent;
