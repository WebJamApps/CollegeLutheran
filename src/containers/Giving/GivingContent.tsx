import React from 'react';
import ELCALogo from '../../components/elcaLogo';

const pageName = 'giving';

const GivingContent = (): JSX.Element => (

  <div className="page-content">
    <div className="container-fluid">
      <p style={{ fontSize: '4pt', margin: '0' }}>&nbsp;</p>
      <div className="material-content elevation3" style={{ maxWidth: '998px', paddingBottom: '-80px', margin: 'auto' }}>
        <h3 style={{ paddingTop: '22px', paddingBottom: '15px' }}>Giving</h3>
        <p>
          Thank you for supporting the ministries of College Lutheran Church through your online giving.
          We now use
          {' '}
          <i>BreezeChMS</i>
          {' '}
          to process online gifts. Giving electronically from your bank account or credit card is easy!
          Just complete the form below to support the ministries of CLC with your offering.
        </p>
        <i>
          Due to technical difficulties, we are transitioning away from our previous online giving processor.
          If you require access to Vanco, click
          {' '}
          <a href="https://gp.vancopayments.com/gpo/#/donateNow/195e5d99-170c-4f3a-b5eb-e61ff13" target="_blank" rel="noreferrer">here</a>
          . For further assistance, email us at
          {' '}
          <a href="mailto:office1@collegelutheran.org">office1@collegelutheran.org</a>
        </i>
      </div>
      <p>&nbsp;</p>
      {/* <div id="breeze_giving_embed" data-subdomain="collegelutheranchurch" /> */}
      <div style={{
        margin: 'auto', textAlign: 'center', padding: '0', width: '100%', marginLeft: '-15px',
      }}
      >
        <iframe
          className="giving-iframe"
          src="https://collegelutheranchurch.breezechms.com/give/online"
          title="Just Giving"
        >
          <p>Your browser does not support iframe.</p>
        </iframe>
      </div>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <ELCALogo pageName={pageName} />
      <p>&nbsp;</p>
    </div>
  </div>
);

export default GivingContent;
