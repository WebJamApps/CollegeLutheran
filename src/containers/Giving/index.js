import React, { Component } from 'react';
import DefaultGivingContent from './GivingContent';
import commonUtils from '../../lib/commonUtils';

export default class Giving extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Giving'); }

  render() { return (<DefaultGivingContent />); }
}
