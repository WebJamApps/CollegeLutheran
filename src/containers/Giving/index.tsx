import React from 'react';
import DefaultGivingContent from './GivingContent';
import commonUtils from '../../lib/commonUtils';

const Giving = (): JSX.Element => {
  commonUtils.setTitleAndScroll('Giving', window.screen.width);
  return (<DefaultGivingContent />);
};
export default Giving;
