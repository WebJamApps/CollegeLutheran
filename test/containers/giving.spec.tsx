import React from 'react';
import { shallow } from 'enzyme';
import Giving from '../../src/containers/Giving/index';
import DefaultGivingContent from '../../src/containers/Giving/GivingContent';

function setup() {
  const props = {};
  const wrapper = shallow(<Giving />);
  return { props, wrapper };
}

describe('Giving', () => {
  it('Renders the Giving component', () => {
    const { wrapper } = setup();
    expect(wrapper.find(DefaultGivingContent)
      .dive()
      .find('div.page-content')
      .exists()).toBe(true);
  });
});
