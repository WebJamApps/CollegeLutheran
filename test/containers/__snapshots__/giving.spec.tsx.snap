// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Giving Renders the Giving component 1`] = `
<div
  style={
    {
      "width": "100%",
    }
  }
>
  <div
    className="MuiBox-root css-1mx15d0"
  >
    <div
      className="MuiBox-root css-ezkpjx"
    >
      <div
        className="material-content elevation3"
        style={
          {
            "margin": "auto",
            "marginLeft": "20px",
            "marginRight": "20px",
            "textAlign": "left",
          }
        }
      >
        <h3>
          Giving
        </h3>
        <p>
          Thank you for supporting the ministries of College Lutheran Church through your online giving. We now use
           
          <i>
            BreezeChMS
          </i>
           
          to process online gifts. Giving electronically from your bank account or credit card is easy! Just complete the form below to support the ministries of CLC with your offering.
        </p>
        <i>
          If you have any questions about online giving, please contact us at (540) 389-4963 or
           
          <a
            href="mailto:office1@collegelutheran.org"
          >
            office1@collegelutheran.org
          </a>
        </i>
      </div>
    </div>
    <div
      className="MuiBox-root css-1v9pxvv"
    >
      <iframe
        className="giving-iframe"
        height="100%"
        src="https://collegelutheranchurch.breezechms.com/give/online"
        style={
          {
            "overflowX": "scroll",
          }
        }
        title="Just Giving"
        width="100%"
      >
        <p>
          Your browser does not support iframe.
        </p>
      </iframe>
    </div>
    <div
      className="MuiBox-root css-t0sx0o"
    >
      <div
        style={
          {
            "marginLeft": "-9px",
            "textAlign": "center",
          }
        }
      >
        <a
          href="http://www.elca.org/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="ELCA LOGO"
            id="elcaLogo"
            src="https://dl.dropboxusercontent.com/s/wkzubcmmm3pqst4/elca-logo.png?dl=0"
            style={
              {
                "margin": "auto",
                "marginLeft": "",
                "paddingTop": "30px",
                "width": "308px",
              }
            }
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;